api/api.go

```diff
@@ -95,7 +96,9 @@ func handleError(ctx context.Context, w http.ResponseWriter, err error, data log
 		case apierrors.ErrRequiredFieldMissing,
 			apierrors.ErrEmptyRequestBody,
 			apierrors.ErrEmptyBookID,
-			apierrors.ErrEmptyReviewID:
+			apierrors.ErrEmptyReviewID,
```

Comment:
I think you have to add the following validation errors as well, which should return `BadRequest`:
```
ErrEmptyReviewUser
ErrLongReviewMessage
```